handler.go

```diff
@@ -239,6 +253,7 @@ type handlerConfig struct {
 	HandleGRPC                   bool
 	HandleGRPCWeb                bool
 	RequireConnectProtocolHeader bool
+	IdempotencyLevel             IdempotencyLevel
```

Comment:
I'd like to expose this on the `Spec` for each method too - you should be able to do this by adding a line to `newSpec` below.

Reply:
Fixed in c2c20b30b364b2c84fd4c753bf5a68cc1dee6291.